docs/api/doc-block-story.md

```diff
@@ -0,0 +1,179 @@
+
+<div class="aside">
+
+Typically you want to use the [`Canvas` block](./doc-block-canvas.md) to render a story with a surrounding border and the source block, but you can use the `Story` block to render just the story.
+
+</div>
+
+![Screenshot of Story block](TK)
+
+<!-- prettier-ignore-start -->
+```md
+{/* ButtonDocs.mdx */}
+import { Meta, Story } from '@storybook/blocks';
+import * as ButtonStories from './Button.stories';
+
+<Meta of={ButtonStories} />
+
+<Story of={ButtonStories.Primary} />
+```
+<!-- prettier-ignore-end -->
+
+## Story
+
+```js
+import { Story } from '@storybook/blocks';
+```
+
+<details>
+<summary>Configuring with props <strong>and</strong> parameters</summary>
+
+ℹ️ Like most blocks, the `Story` block is configured with props in MDX. Many of those props derive their default value from a corresponding [parameter](../writing-stories/parameters.md) in the block's namespace, `parameters.docs.story`.
+
+The following `autoplay` configurations are equivalent:
+
+```js
+// Button.stories.js
+export const Basic {
+  parameters: {
+    docs: {
+      story: { autoplay: true },
+    },
+  },
+};
+```
+
+<!-- prettier-ignore-start -->
+```md
+{/* ButtonDocs.mdx */}
+<Story of={ButtonStories.Basic} autoplay />
+```
+<!-- prettier-ignore-end -->
+
+The example above applied the parameter at the [story](../writing-stories/parameters.md#story-parameters) level, but it could also be applied at the [component](../writing-stories/parameters.md#component-parameters) (or meta) level or [project](../writing-stories/parameters.md#global-parameters) level.
+
+</details>
+
+### `autoplay`
+
+Type: `boolean`
+
+Default: `parameters.docs.story.autoplay`
+
+Determines whether a story's play function runs.
+
+Because all stories render simultaneously in docs entries, play functions can perform arbitrary actions that can interact with each other (such as stealing focus or scrolling the screen). For that reason, by default, stories **do not run play functions in docs mode**.
+
+However, if you know your play function is “safe” to run in docs, you can use this prop to run it automatically.
+
+### `height`
+
+Type: `string`
+
+Default: `parameters.docs.story.height`
+
+Set a minimum height (note for an iframe this is the actual height) when rendering a story in an iframe or inline. This overrides `parameters.docs.story.iframeHeight` for iframes.
+
+### `inline`
+
+Type: `boolean`
+
+Default: `parameters.docs.story.inline` or `true` (for [supported frameworks](../configure/frameworks-feature-support.md))
+
+Determines whether the story is rendered `inline` (in the same browser frame as the other docs content) or in an iframe.
+
+### `meta`
+
+Type: CSF file exports
+
+Specifies the CSF file to which the story is associated.
+
+You can render a story from a CSF file that you haven’t attached to the MDX file (via `Meta`) by using the `meta` prop. Pass the **full set of exports** from the CSF file (not the default export!).
```

Comment:
My thought here was that it's better to start the sentence with the "why" rather than the "how". Especially since the "how" is already implied by this paragraph being under the `meta` heading.

But if you feel strongly, I'm happy to update it.

Reply:
Let's leave it as is, for now. We can improve it later on.